website/pages/docs/block.mdx

```diff
@@ -6,6 +6,22 @@ description: 'Million.js introduces the concept of a Block. Blocks are a way for
 import { Callout } from 'nextra-theme-docs';
 import { AutomaticModeWarning } from '../../components/automatic-mode-warning';
 
+## What's a block?
+
+Million.js is a library that enables you to create _blocks_. A block is a special [Higher Order Component (HOC)](https://legacy.reactjs.org/docs/higher-order-components.html) that can be used as a React component but is hyper-optimized for rendering speed.
```

Comment:
The first sentence here is created lower down. If this is a reference doc, then you don't have to talk about how Million lets you create blocks.

```suggestion
A block is a specialized [Higher Order Component (HOC)](https://legacy.reactjs.org/docs/higher-order-components.html) that functions as a React component but more optimized for rendering speed.
```

Reply:
Thanks for this again, Sarah! I really appreciate it.